SECURITY.md

```diff
@@ -0,0 +1,6 @@
+# Security Policy
+
+Security issues and bugs should be reported privately, via email, to the Microsoft Security Response Center (MSRC) <secure@microsoft.com>. You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Further information, including the MSRC PGP key, can be found in the [Security TechCenter](https://www.microsoft.com/msrc/faqs-report-an-issue).
```

Comment:
please add the "Reporting a Vulnerability" heading (https://github.com/dotnet/runtime/blob/master/SECURITY.md#reporting-a-vulnerability).  Otherwise LGTM